conf/nginx/templates/nginx.conf.web.clamav.signature.provider.template

```diff
@@ -0,0 +1,13 @@
+# Carbonio ClamAV Signature Provider proxy default configuration
+#
+server
+{
+    listen 127.78.0.1:10000;
```

Comment:
Hey @keshavbhatt consider also defining an entire new service with dedicated ip port, for example:
- carbonio-signatures on 127.78.0.30:10000.

This because 127.78.0.1 is reserved for carbonio-proxy, port 443.

Reply:
Let's discuss this in the call we are having tomorrow.

Reply:
In meeting we decided to create a seperate consul service to handle exposer.